README.md

```diff
@@ -26,8 +27,10 @@
   <a href="https://www.codetriage.com/romcal/romcal" target="_blank" rel="noopener noreferrer"><img alt="Code Triage Helpers" src="https://www.codetriage.com/romcal/romcal/badges/users.svg" /></a>
 </p>
 
-> **Note**
-> This project is still in beta (using the `dev` npm distribution tag) until it reaches final version 3.0.0. There could be breaking changes between minor versions. Please refer to the [changelog](CHANGELOG.md) for significant updates and breaking changes.
+> [!NOTE]
+> This project is still in beta (using the `dev` npm distribution tag) until it reaches final version 3.0.0.
+> There could be breaking changes between minor versions.
```

Comment:
There should be no breaking changes between minor versions, as we are targeting `v3.0.1` (although the `dev` versions contain `3.0.0`, e.g. `3.0.0-dev.97`) and currently incrementing the dev suffix only (currenly `97`).

Reply:
I didn't change the text, only the line breaks.